docs/release-notes/self-hosted/self-hosted-v5.0.0.md

```diff
@@ -0,0 +1,118 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Self-hosted v5.0.0.
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/3.9.16
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/4.0.28
+---
+
+# Self-hosted v5.0.0
+
+These release notes are for [Codacy Self-hosted v5.0.0](https://github.com/codacy/chart/releases/tag/5.0.0){: target="_blank"}, released on December 14, 2021. <!-- TODO Update release date -->
+
+To upgrade Codacy, follow [these instructions](../../chart/maintenance/upgrade.md).
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Breaking changes
+
+This version of Codacy Self-hosted introduces the following breaking changes:
+
+-   Dropped support for legacy manual organizations.
+
+    !!! important
+        **If you're using legacy manual organizations** please contact <mailto:support@codacy.com> before upgrading to Codacy Self-hosted 5.0.0 or above so that <span class="skip-vale">we</span> can assist you with migrating your legacy organizations.
+
+-   The Codacy components `codacy-activities`, `hotspots-api`, and `hotspots-worker` are no longer needed and were removed.
```

Comment:
are we always referring to these as components or can we change this to services?

I'm cool with components, but services seems more self explanatory (but only if users are not used to components as the terminology already)

Reply:
In the documentation we generally refer to "components":

https://docs.codacy.com/chart/requirements/#:~:text=components

I'm also concerned that "services" already has a well-defined meaning in the Kubernetes lingo:

https://kubernetes.io/docs/concepts/services-networking/service/

Reply:
Ok, then go with "components" then! 👍